app/src/main/java/com/example/vehicleinfocheck/Element.java

```diff
@@ -1,9 +1,10 @@
 package com.example.vehicleinfocheck;
+// An element is a field present in the website: an input field, a button, etc.
 
 import android.util.Log;
 
-// An element is a field present in the website: an input field, a button, etc.
 // This class defines all methods pertaining to elements
+// However, this class is not used in this project as the required methods did not work
```

Comment:
Move this comment to first line of this file indicating this file is not part of this project!